### ISSUE TITLE
issue implementing it on my own robot

### ISSUE THREAD

**Issue by rs73366** [rafaelbarretorb/rrt_star_global_planner#16]
HI there so i have been trying to run your package on my robot, if i am correct these are the steps i have followed. 
I have already gone through tutorials [1](http://wiki.ros.org/navigation/Tutorials/Writing%20A%20Global%20Path%20Planner%20As%20Plugin%20in%20ROS) & [2](https://github.com/ros-planning/navigation) as  mentioned by you but my exact steps are  

1. cloned your repository in my catkin_ws
2.went into directory of **my robot's launch folder** and opened the move_base.launch file
3. edited the file as shown
4. `<launch>
  
  <arg name="cmd_vel_topic" default="/cmd_vel" />
  <arg name="odom_topic" default="odom" />
  <arg name="move_forward_only" default="false"/>
  <arg name="base_global_planner" default="rrt_star_global_planner/RRTStarPlanner"/>

  
  
  <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
    
    
    <param name="base_local_planner" value="dwa_local_planner/DWAPlannerROS" />
    <rosparam file="$(find xbotversionten_navigation)/parameter_files/costmap_common_params.yaml" command="load" ns="global_costmap" />
    <rosparam file="$(find xbotversionten_navigation)/parameter_files/costmap_common_params.yaml" command="load" ns="local_costmap" />
    <rosparam file="$(find xbotversionten_navigation)/parameter_files/local_costmap_params.yaml" command="load" />
    <rosparam file="$(find xbotversionten_navigation)/parameter_files/global_costmap_params.yaml" command="load" />
    <rosparam file="$(find xbotversionten_navigation)/parameter_files/move_base_params.yaml" command="load" />
    <rosparam file="$(find xbotversionten_navigation)/parameter_files/teb_local_planner_params.yaml" command="load" />
    <rosparam file="$(find xbotversionten_navigation)/parameter_files/rrt_star_params.yaml" command="load"/>
    <remap from="cmd_vel" to="$(arg cmd_vel_topic)"/>
    <remap from="odom" to="$(arg odom_topic)"/>
    <param name="TEBPlannerROS/min_vel_x" value="0.0" if="$(arg move_forward_only)" />
  </node>
</launch>`

I also pasted the rrt_star_params.yaml file in parameter_files(aka config) of my robot's folder. 
after launching my robot as

`roslaunch xbotversionten_navigation xbotversionten_navigation.launch`    i get the following error

`RLException: error loading <rosparam> tag: 
        file /home/rahul/catkin_ws/src/xbotversionten_navigation/parameter_files/rrt_star_params.yaml contains invalid YAML:
mapping values are not allowed here
  in "<string>", line 3, column 15:
    RRTStarPlanner:
                  ^
XML is <rosparam command="load" file="$(find xbotversionten_navigation)/parameter_files/rrt_star_params.yaml"/>
The traceback for the exception was written to the log file`

can you look into it ?

**Comment by lgl-960709** [comment]
> 1. move_base.启

Have you solved this problem?

**Comment by rs73366** [comment]
> 

No, I have instead starting to look how it relates to global planner tutorial mentioned. although the global planner tutorial works.
